blocks/empty.pfb

```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Empty",
+    "title": "Code",
```

Comment:
Why this change ?

Reply:
For a user, when right-clicking on the interface they had the choice between "Empty Block" and "Markdown Block", I replaced Empty with Code for clarity purposes